spec/services/alerts_service_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'rails_helper'
+
+describe AlertsService do
+  shared_examples 'a threshold that needs to be alerted' do
+    it 'sends an email' do
+      allow(AlertsService.new).to receive(:call)
+
+      expect {
+        subject
+      }.to change(ActionMailer::Base.deliveries, :count).by(1)
+    end
+
+    it 'updates last sent date to current day' do
+      subject
+
+      expect(alert.reload.last_sent_date). to eq(Time.zone.today)
+    end
+  end
+
+  shared_examples 'a threshold that does not needs to be alerted' do
```

Comment:
shouldn't this be
```suggestion
  shared_examples 'an alert that does not need to be triggered' do
```
?